TCAT/Model/Place.swift

```diff
@@ -13,7 +13,7 @@ enum PlaceType: String, Codable {
     case applePlace, busStop, unknown
 }
 
-@objc(Place) class Place: NSObject, Codable {
+class Place: NSObject, Codable {
```

Comment:
Let's make sure Today Extension & Siri Shortcuts still works cuz it seems like it may have been tied to that. Was added in this [commit](https://github.com/cuappdev/ithaca-transit-ios/commit/099bb2cec5c90453720883e43ad92ab81b546fbc) cc @ywsang